src/vl-map.src.js

```diff
@@ -0,0 +1,45 @@
+import { define, awaitScript } from '/node_modules/vl-ui-core/vl-core.js';
+import { VlMap } from "./vl-map";
+import { VlMapOverviewMap } from "./vl-map-overview-map";
+import { VlMapLayer } from "./vl-map-layer";
+import { VlMapBaseLayer } from "./vl-map-baselayer";
+import { VlMapBaseLayerGRBGray } from "./vl-map-baselayer-grb-gray";
+import { VlMapBaseLayerGRB } from "./vl-map-baselayer-grb";
+import { VlMapBaseLayerGRBOrtho } from "./vl-map-baselayer-grb-ortho";
+import { VlMapAction } from "./vl-map-action";
+import { VlMapSelectAction } from "./vl-map-select-action";
+import { VlMapLayerStyle } from "./vl-map-layer-style";
+import { VlMapLayerCircleStyle } from "./vl-map-layer-circle-style";
+import { VlMapSearch } from "./vl-map-search";
+
+Promise.all([
+    awaitScript('vl-map-openlayers', '/node_modules/vl-mapactions/lib/openlayers/dist/ol.js'),
+    awaitScript('vl-map-proj4', '/node_modules/proj4/dist/proj4.js'),
+    awaitScript('vl-map-mapactions', '/node_modules/vl-mapactions/dist/mapactions-min.js')]
```

Comment:
Dit kan waarschijnlijk best herschreven worden als gewone imports. Dan akn de replace zijn werk doen in de paden.